.github/workflows/build_and_deploy.yml

```diff
@@ -3,6 +3,8 @@ name: build-and-deploy
 
 on:
   push:
+    tags-ignore:
+      - '*'
```

Comment:
Had to follow-up in https://github.com/vercel/next.js/pull/87207 and https://github.com/vercel/next.js/commit/3884de9b4ef7f90d5ec451302c4b08c287a7223c.

Just `tags`/`tags-ignore` implies all branch pushes are ignored. So now we only list `branches` which implies all tag pushes are ignored.

Reply:
Works now: https://github.com/vercel/next.js/actions/runs/20244398106 -> https://github.com/vercel/next.js/actions/runs/20244500473

Only build-and-deploy for branch. None for tags:
<img width="2538" height="616" alt="CleanShot 2025-12-15 at 20 24 21@2x" src="https://github.com/user-attachments/assets/e3e99293-fbe8-42b2-8fdf-0c00ea5b9324" />